### PR TITLE
feat: add initial `test5.gno.land` deployment

### LINKED ISSUE #3061: Test5 release
## Description

Launch Test5 on November 12, 2024

### PR DESCRIPTION
**Pull request by zivkovicmilos**
## Description

This PR adds the initial `test5.gno.land` deployment params.

List of transaction changes as opposed to current `master` examples:
https://github.com/gnolang/gno/commit/2e9f5ce8ecc90ee81eb3ae41c06bab30ab926150

- **131 txs in the `genesis.json`**

All validator entities added:
- @r3v4s ✅ 
- @D4ryl00 ✅ 
- @sw360cab ✅ 
- @mazzy89 ✅ 
- @n0izn0iz ✅ 
- @albttx ✅ 

Release we will use for the initial `test5.gno.land` deployment:
https://github.com/gnolang/gno/pkgs/container/gno%2Fgnoland/303668315?tag=chain-test5.0

Closes #3061 

<details><summary>Contributors' checklist...</summary>

- [x] Added new tests, or not needed, or not feasible
- [x] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [x] Updated the official documentation or not needed
- [x] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
</details>

### CODE REVIEW

File: misc/deployments/test5.gno.land/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- `p2p.laddr` - the listen address for P2P traffic, **specific to every node deployment**. It is advised to use a
+  reverse-proxy, and keep this value at `tcp://0.0.0.0:<port>`.
+- `p2p.max_num_outbound_peers` - the max number of outbound peer connections. **Advised to be `40`**.
+- `p2p.persistent_peers` - the persistent peers. ⚠️ **Required to be
+  `TODO`** ⚠️.
+- `p2p.pex` - if using a sentry node architecture, should be `false`. **If not, please set to `true`**.
+- `p2p.external_address` - the advertised peer dial address. If empty, will use the same port as the `p2p.laddr`. This
+  value should be **changed to `{{ your_ip_address }}:26656`**
+- `p2p.flush_throttle_timeout` - the timeout for flushing multiplex data. ⚠️ **Required to be `10ms`** ⚠️.
+- `p2p.peer_gossip_sleep_duration` - the timeout for peer gossip. ⚠️ **Required to be `10ms`** ⚠️.
```
Comment (r3v4s) [inline misc/deployments/test5.gno.land/README.md:42]:
```suggestion
- `consensus.peer_gossip_sleep_duration` - the timeout for peer gossip. ⚠️ **Required to be `10ms`** ⚠️.
```

typo

Comment (zivkovicmilos) [reply misc/deployments/test5.gno.land/README.md]: @sw360cab caught it as well, we fixed it in his PR 🙏

**Review by albttx — APPROVED** [review]

**Review by r3v4s — APPROVED** [review]
LGTM 👍

**Review by sw360cab — APPROVED** [review]

**Comment by zivkovicmilos** [comment]
@moul 
I'm merging this PR now, we've verified the deployment and releases ✅

**Pull request merged**